src/frontend/components/Felleskomponenter/SystemetLaster/SystemetLaster.tsx

```diff
@@ -1,30 +1,13 @@
 import React from 'react';
 
-import styled from 'styled-components';
-
-import { BodyShort, Loader } from '@navikt/ds-react';
-
-const StyledLoader = styled(Loader)`
-    margin-top: 3rem;
-    height: 10rem;
-    width: 10rem;
-`;
-
-const StyledMain = styled.main`
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
-    height: 100vh;
-    width: 100vw;
-`;
+import { BodyShort, Loader, VStack } from '@navikt/ds-react';
 
 const SystemetLaster = () => {
     return (
-        <StyledMain>
```

Comment:
Kan fortsatt være et poeng å ha `<main> ... </main>`her

Reply:
Ettersom AppContainer allerede har en main i seg og SystemetLaster tas i bruk i denne komponenten, fjernet jeg main slik at det ikke ble dobbelt opp.

Men dersom SystemetLaster blir tatt i bruk gjennom DisabledApp, så er det ingen main som wrappes rundt. Så om jeg legger til main inni eller ikke så vil det føre feil syntaks med mindre andre endringer gjøres for å fikse dette.

Jeg endrer det tilbake slik det var fra før.

Reply:
Mulig du ikke trenger `<main />` her om det nå ligger i `DisabledApp` og `AppContainer`? Virker litt rart å ha en `<main />` tag her inne